go.mod

```
@@ -14,14 +14,15 @@ require (
 	github.com/gorilla/websocket v1.5.0
 	github.com/grafana/xk6-redis v0.1.1
 	github.com/grafana/xk6-timers v0.1.2
-	github.com/grafana/xk6-websockets v0.1.3
+	github.com/grafana/xk6-websockets v0.1.4-0.20220909124839-39d27946bc0c
```

Maybe create a tag in `github.com/grafana/xk6-websockets`? :thinking:

---

It doesn't sound optimal to me to tag an under-development branch. I expect you want it for documenting better, is it right?

Is a comment a good alternative?

```suggestion
	github.com/grafana/xk6-websockets v0.1.4-0.20220909124839-39d27946bc0c  // v0.1.3 plus time series migration
```

---

Ah no, I think it's fine even without :)